setup.py

```diff
@@ -43,7 +43,7 @@ def version(path):
 minimal_requirement = [
     "click>=8.0.0",
     "pydantic",
-    "SQLAlchemy",
+    "SQLAlchemy>=1.4",
```

Comment:
Sometimes we also need to update requirements.txt when changing setup.py, but seems like the requirements.txt is already compliant
https://github.com/LineaLabs/lineapy/blob/main/requirements.txt#L42